src/client/game.cpp

```diff
@@ -1193,8 +1194,8 @@ void Game::run()
 
 		// Calculate dtime =
 		//    m_rendering_engine->run() from this iteration
-		//  + Sleep time until the wanted FPS are reached
-		draw_times.limit(device, &dtime);
+		//  + Sleep time until the wanted tick rate are reached
+		draw_times.limit(device, &dtime, 60);
```

Comment:
Why the hardcoded 60?

Reply:
Make this a setting is in the TODO list for the PR.

Reply:
And 60 is the default FPS cap in MT